recipes/openssl/3.x.x/conanfile.py

```diff
@@ -610,6 +610,7 @@ def build(self):
                 self._create_targets()
                 with self._make_context():
                     self._make()
+                    self.run("perl source_subfolder/configdata.pm --dump")
```

Comment:
what is it?

Reply:
yes, it's code to dump OpenSSL configuration

Reply:
check this out: https://github.com/openssl/openssl/blob/master/Configure#L32-L48

Reply:
It was debug code from us testing it

Reply:
it might be useful for troubleshooting in future, e.g. debugging bugs, it's worth keeping